tests/test_services.py

```diff
@@ -26,3 +30,18 @@ def level_name(message):
         # Request times below 3 secs are set as info
         status_message = SUCCESS%2.32
         eq_('info', level_name(status_message))
+
+    def test_init(self):
+        # Test that ServiceStatus can create an Authenticator.
+        with temp_config() as config:
+            config[Configuration.POLICIES] = {
+                Configuration.AUTHENTICATION_POLICY: {
+                    "providers": [
+                        {"module": 'api.millenium_patron',
+                         Configuration.URL: "http://url"}
```

Comment:
This will work since you're not testing anything for real, but I would use api.mock_authentication here.